mava/wrappers/saveable.py

```diff
@@ -0,0 +1,34 @@
+from typing import Any, Dict
+
+from acme.core import Saveable as AcmeSaveable
+
+"""SaveableWrapper as needed to use the Acme JAX checkpointer."""
+
+
+class SaveableWrapper(AcmeSaveable):
+    def __init__(self, state: Dict[str, Any]):
+        """Initialise system state
+
+        Args:
+            state (_type_):  system state represented by a dictionary of saved variables
```

Comment:
This is a bit unintuitive? Should it be a dictionary of variables to save?

Reply:
Changed